pallets/offchain-worker/src/lib.rs

```diff
@@ -252,6 +257,19 @@ pub mod pallet {
 						let mut source_index = 0;
 						for source in &urls::DATA_SOURCE_LIST {
 							let task_index = urls::TOTAL_DATA_SOURCE_NUMBER * account_index + source_index;
+
+							// SBP M1 review: if I understand correctly, the logic is "distributing" the work
+							// over the various offchain workers, and a given account requesting a claim will
+							// always be processed by only 1 offchain worker.
+							// 
+							// This can be a problem because:
+							// 1) as mentionned in offchain_worker function, some claims might be erased before being processed.
+							// 2) there is therefore no voting / averaging system in place if every account is processed by at most 1 ocw.
+							//
+							// The OCW documentation mentions the following:
+							// "Note that the results from off-chain workers are not subject to regular transaction verification.
+							//  A verification mechanism (e.g. voting, averaging, checking sender signatures, or simply "trusting")
+							//  should be implemented to determine what information gets into the chain."
```

Comment:
we designed the mechanism similar to voting, or like frequence most chosen method after multiple off-chain workers submit the query result.